MANIFEST.in

```diff
@@ -5,6 +5,7 @@ include LICENSE
 include CONTRIBUTING.md
 include versioneer.py
 include metpy/_version.py
+include metpy/registry.txt
```

Comment:
How about something more descriptive--maybe `static-data-manifest.txt`?

Reply:
Yeah, that would be better. @jrleeman should I fix this or are you gonna edit this branch?

Reply:
Editing now 😄